src/js/tech/html5.js

```diff
@@ -321,7 +321,7 @@ class Html5 extends Tech {
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
         this.one('webkitendfullscreen', function() {
-          this.trigger('fullscreenchange');
+          this.trigger('fullscreenchange', 'not-fullscreen');
```

Comment:
Do we want to balance this, so that `handleTechFullscreenChange` is always setting player.isFullscreen, instead of just on exit?